packages/ui/package.json

```diff
@@ -38,14 +38,16 @@
     "prune": "ts-prune src"
   },
   "dependencies": {
-    "@dnd-kit/core": "^5.0.1",
-    "@dnd-kit/sortable": "^6.0.0",
+    "@dnd-kit/core": "^6.0.8",
+    "@dnd-kit/sortable": "^7.0.2",
     "@heroicons/react": "^2.0.13",
     "@scena/react-guides": "^0.16.0",
+    "antd": "^5.9.4",
+    "form-render": "^2.2.16",
```

Comment:
I've introduced a library called 'FormRender' for customizing the propPanel. I've also added 'Ant Design,' which is a dependency of FormRender. 

Using FormRender allows us to efficiently create UI, and the  Ant Design will make it much easier to customize the UI going forward. 
Ant Design allows for customization through [Design Tokens and Component Tokens ](https://ant.design/docs/react/customize-theme), so I believe this will significantly simplify achieving our future goals for Custom Design for UI #243.

FormRender: https://xrender.fun/form-render
Ant Design: https://ant.design/